package.json

```diff
@@ -31,25 +31,25 @@
     "axios": "^0.21.1",
     "babel-plugin-styled-components": "^1.10.7",
     "dedent": "^0.7.0",
-    "eslint": "^6.8.0",
-    "gatsby": "^2.24.78",
-    "gatsby-awesome-pagination": "^0.3.6",
+    "eslint": "^7.23.0",
+    "gatsby": "^3.2.1",
+    "gatsby-awesome-pagination": "^0.3.8",
     "gatsby-cli": "^3.2.0",
-    "gatsby-image": "^2.2.34",
+    "gatsby-image": "^3.2.0",

```

Comment:
Isn't the `gatsby-image` plugin changed to `gatsby-plugin-image`? AFAIR, it is and that needs to be changed as well.
/cc @Tanuj22

Reply:
Yes, I had dropped a msg to @leecalcote regarding this and was waiting for his response for this. 
Along with this, we also need to Update <Img> with fixed and fluid params as they are also deprecated
I had referred this [Guide](https://www.gatsbyjs.com/docs/reference/release-notes/image-migration-guide/)

Reply:
You can go ahead, and test out the needed changes.

Reply:
Yeah. Then we are going to have to replace all the <img> tags with <StaticImage> from gatsby.

Reply:
@Nikhil-Ladha should we keep this in a separate brach until all the changes are made?

Reply:
Later on, how do we merge that branch into the latest one. (Might be a silly question, but I am not sure about the process).
Or we would just do a cut paste of the complete code?

Reply:
We can create a branch from master say `sitev3` and then merge this PR there. Then all the changes can be made on the `sitev3` branch. Then when everthing works open a PR from `sitev3` to master

Reply:
Good, but I doubt if there would a lot of conflicts arising at that time while making the PR?

Reply:
But, if that's something which won't be there or we are confident about that, then it would a better approach for sure.

Reply:
Yes @Tanuj22 , we would need to change those too but instead of `<StaticImage>` we would be using `<GatsbyImage>` as it would cater if we need Dynamic Images too in the future

Reply:
For static image `<StaticImage>` will be used (those that are not sourced from graphql)